File: 22.10_PS/1002_수_정렬하기.py
# boj. 2750
# 버블정렬

import sys

input = sys.stdin.readline
n = int(input())
arr = []

for i in range(n):
    arr.append(int(input()))

def swap(arr, x, y):
    if arr[x] > arr[y]:
        arr[x], arr[y] = arr[y], arr[x]
    return arr

for i in range(n, 1, -1):
    for j in range(i - 1):
        arr = swap(arr, j, j + 1)

for i in arr:
    print(i)

File: 22.10_PS/1002_수_정렬하기.java
// boj. 2750
// 버블정렬

import java.io.*;
import java.util.*;

public class Main {
    static int[] arr;
    public static void ma기n(String args[]) throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken());
        arr = new int[n];

        for (int i = 0; i < n; i++) {
            st = new StringTokenizer(br.readLine());
            arr[i] = Integer.parseInt(st.nextToken());
        }

        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - 1 - i; j++) {
                Swap(j, j+1);
            }
        }

        for (int i = 0; i < n; i++) {
            System.out.println(arr[i]);
        }

    }

    static void Swap(int i, int j) {
        int tmp;
        if (arr[i] > arr[j]){
            tmp = arr[i];
            arr[i] = arr[j];
            arr[j] = tmp;
        }
    }
}